gradle/libs.versions.toml

```diff
@@ -0,0 +1,66 @@
+sonar-analyzer-test-commons = { module = "org.sonarsource.analyzer-commons:sonar-analyzer-test-commons", version.ref = "sonar-commons" }
+sonar-plugin-api = { module = "org.sonarsource.api.plugin:sonar-plugin-api", version.ref = "sonar-plugin-api" }
+sonar-plugin-api-impl = { module = "org.sonarsource.sonarqube:sonar-plugin-api-impl", version.ref = "sonarqube" }
+sonar-plugin-api-test-fixtures = { module = "org.sonarsource.api.plugin:sonar-plugin-api-test-fixtures", version.ref = "sonar-plugin-api" }
+sonar-ws = { module = "org.sonarsource.sonarqube:sonar-ws", version.ref = "sonarqube" }
+sonar-orchestrator = { module = "org.sonarsource.orchestrator:sonar-orchestrator", version.ref = "sonar-orchestrator" }
+sonar-orchestrator-junit5 = { module = "org.sonarsource.orchestrator:sonar-orchestrator-junit5", version.ref = "sonar-orchestrator" }
+slf4j-api = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
+junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit" }
+junit-jupiter-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "junit" }
+assertj-core = { module = "org.assertj:assertj-core", version.ref = "assertj" }
+mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+jfrog-buildinfo-gradle = { module = "org.jfrog.buildinfo:build-info-extractor-gradle", version.ref = "jfrog-gradle" }
+
+# newly added elements
+sonar-lint = { module = "org.sonarsource.sonarlint.core:sonarlint-core", version.ref = "sonar-lint" }
+sslr = { module = "org.sonarsource.sslr:sslr-core", version.ref = "sslr" }
+sslr-test = { module = "org.sonarsource.sslr:sslr-testing-harness", version.ref = "sslr" }
+archunit = { module = "com.tngtech.archunit:archunit-junit5", version.ref = "archunit" }
+snakeyaml = { module = "org.snakeyaml:snakeyaml-engine", version.ref = "snakeyaml" }
+minimaljson = { module = "com.eclipsesource.minimal-json:minimal-json", version.ref = "minimaljson" }
+
+# copied elements from sonar-text with no found reference in sonar-iac
+sonar-java-checks = { module = "org.sonarsource.java:java-checks", version.ref = "sonar-java" }
+sonar-scanner-gradle = { module = "org.sonarsource.scanner.gradle:sonarqube-gradle-plugin", version.ref = "sonar-scanner-gradle" }
+jackson-dataformat-yaml = { module = "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml", version.ref = "jackson" }
+com-networknt-jsonSchemaValidator = { module = "com.networknt:json-schema-validator", version.ref = "json-schema-validator" }
+diffplug-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless-gradle" }
+diffplug-blowdryer = { module = "com.diffplug:blowdryer", version.ref = "blowdryer-gradle" }
+sonar-license-api = { module = "com.sonarsource.license:license-api", version.ref = "sonar-license-api" }
+yguard = { module = "com.yworks:yguard", version.ref = "yguard"}
+eclipse-jgit = { module = "org.eclipse.jgit:org.eclipse.jgit", version.ref = "eclipse-jgit" }
```

Comment:
Some of those ependencies are not needed, e.g. jgit. please clean up

Reply:
Yes, for now this file is a copy from the one in SonarText + few additions I made.
I plan to clean it + also check if all dependencies put everywhere are necessary.